Source/NSCell.m

```diff
@@ -443,43 +445,43 @@ - (void) setFloatValue: (float)aFloat
 
 /**
  *<p>Sets the NSCell's value to anInt.</p>
- *<p>See Also: -intValue</p> 
+ *<p>See Also: -intValue</p>
  */
 - (void) setIntValue: (int)anInt
 {
   NSNumber *number;
 
-  // NB: GNUstep can set an int value for an image cell. 
+  // NB: GNUstep can set an int value for an image cell.
 
   number = [NSNumber numberWithInt: anInt];
   [self setObjectValue: number];
 }
 
 /**
  *<p>Sets the NSCell's value to anInt.</p>
- *<p>See Also: -integerValue</p> 
+ *<p>See Also: -integerValue</p>
  */
 - (void) setIntegerValue: (NSInteger)anInt
 {
   NSNumber *number;
 
-  // NB: GNUstep can set an int value for an image cell. 
+  // NB: GNUstep can set an int value for an image cell.
 
   number = [NSNumber numberWithInteger: anInt];
   [self setObjectValue: number];
 }
 
-/**<p>Sets the cell's value to a NSString. 
+/**<p>Sets the cell's value to a NSString.
    The NSCell's type is set to NSTextCellType if needed</p>
-   <p>See Also: -stringValue</p> 
+   <p>See Also: -stringValue</p>
  */
 - (void) setStringValue: (NSString*)aString
 {
   /* We warn about nil for compatibiliy with MacOS X, which refuses
      nil.  */
   if (aString == nil)
     {
-      NSDebugMLLog (@"MacOSXCompatibility", 
+      NSDebugMLLog (@"MacOSXCompatibility",
                     @"Attempt to use nil as string value");
```

Comment:
As there is cleanup here anyway, how about we say `macOS would not allow nil as a string value, proceeding anyway.`?

(Would be ideal if the other OPENSTEPish platform was named fewer times in the code, though.)

Reply:
As this is completely unrelated to my changes I will postpone any changes here.